app/datatables/repository_datatable.rb

```diff
@@ -158,6 +158,7 @@ def get_raw_records
                                  )
                                  .joins(:created_by)
                                  .where(repository: @repository)
+      return @assigned_rows if params[:assigned] == 'assigned'
```

Comment:
I think the condition should be `@my_module && params[:assigned] == 'assigned'`, just like on line 185, and you can remove line 185, since you do this here.